java/libphonenumber/src/com/google/i18n/phonenumbers/MetadataManager.java

```diff
@@ -61,19 +61,15 @@
   private MetadataManager() {
   }
 
-  private static void close(InputStream in) {
-    if (in != null) {
-      try {
-        in.close();
-      } catch (IOException e) {
-        LOGGER.log(Level.WARNING, e.toString());
-      }
-    }
-  }
+  // The size of the byte buffer in bytes used to convert a stream containing metadata for a single
+  // region, to a nanoproto-compatible CodedInputByteBufferNano. This was determined by the size of
+  // the metadata with which we build our actual jars.
```

Comment:
How do you mean the size of the metadata? what does the 16 represent, what does the 1024 represent? Similarly below: why 256? It looks like a calculation but I'm confused what it is calculating.